sdk/lusid/utilities/api_client_builder.py

```diff
@@ -10,11 +10,13 @@
 class ApiClientBuilder:
 
     @staticmethod
-    def build(api_secrets_filename=None, okta_response_handler=None, api_configuration=None):
+    def build(api_secrets_filename=None, okta_response_handler=None, api_configuration=None,
```

Comment:
`certificate_filename` and `proxy_url` should be on `ApiConfiguration` rather than passed in

Reply:
also don't you need the proxy username/password/